AppCenter/AppCenter/MSACAppCenter.h

```diff
@@ -68,76 +69,45 @@
  *
  * @param services Array of services to start.
  */
-+ (void)startFromLibraryWithServices:(NSArray<Class> *)services;
++ (void)startFromLibraryWithServices:(NSArray<Class> *)services NS_SWIFT_NAME(startFromLibrary(services:));
 
 /**
- * Check whether the SDK has already been configured or not.
- *
- * @return YES if configured, NO otherwise.
+ * The flag indicates whether the SDK has already been configured or not.
  */
-+ (BOOL)isConfigured;
+@property(class, atomic, readonly, getter=isConfigured) BOOL configured;
```

Comment:
The following is arguable but there's actually something tricky in the readonly swift properties. I realized it recently but a "readonly" boolean Swift property **expressed** as an adjective will keep the `is` 😅. At least that's my interpretation of this line: [Uses of Boolean methods and properties should read as assertions about the receiver when the use is nonmutating](https://swift.org/documentation/api-design-guidelines/). Examples: [isContinuous](https://developer.apple.com/documentation/appkit/nscell/1529179-iscontinuous), [isKeyWindow](https://developer.apple.com/documentation/uikit/uiwindow/1621612-iskeywindow), [isCanceled](https://developer.apple.com/documentation/sirikit/inridecompletionstatus/2132104-iscanceled), [isFileURL](https://developer.apple.com/documentation/foundation/nsurl/1408782-isfileurl)...
I could find counter examples basically multiple bool properties of [NSCell](https://developer.apple.com/documentation/appkit/nscell) though 😬.

Reply:
Agree, the thing is it's not needed to add the `NS_SWIFT_NAME`, afaik uses the `getter` name by default. So this one is going to be `isConfigured` in swift.

Reply:
Example:
defined as [`@property(class, readonly, nonatomic, getter=isDebuggerAttached) BOOL debuggerAttached;`](https://github.com/microsoft/appcenter-sdk-apple/pull/2201/files#diff-8048fe5774b68eb770d07695708da3b4c6d67cc0a6028c926a1cc7365301ec03R144)

used in swift file as [`AppCenter.isDebuggerAttached`](https://github.com/microsoft/appcenter-sdk-apple/pull/2201/files#diff-95fe3352dd1452ff16b427124eff0d0d891e7b76ae73f2888e2a17c5756b515aR33)

Reply:
Oh interesting, well perfect then :).